### PR TITLE
Modified data type of product id from string to int

### PR DESCRIPTION
**Pull request by rutvik2151**

### CODE REVIEW

File: service/models.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class OrderItem(db.Model):
     # Order Item Table Schema
     ##################################################
     item_id = db.Column(db.Integer, primary_key=True)
-    product = db.Column(db.String(40), nullable=False)
```
Comment (shivkhattar) [inline service/models.py:24]: @rutvik2151 It would be better to rename the column to ```product_id```.

**Review by shivkhattar — CHANGES_REQUESTED** [review]

File: service/models.py (hunk 1 above)
Comment (rutvik2151) [reply service/models.py:24]: Will do.

**Review by shivkhattar — APPROVED** [review]

**Comment by shivkhattar** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/d97e2552-e5d6-4314-93cd-4603445a8066?env_id=ibm:yp:us-south "orders-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/de2c1c4b-d454-4678-8c83-87825f7d8ab3 "order-pipeline") deployed [orders](https://github.com/NYU-Devops-2020-Orders-Team/orders/releases/tag/deploy-ibm-yp-us-south-sk8325-nyu.edu-dev-20201103-194725 "deploy-ibm-yp-us-south-sk8325-nyu.edu-dev-20201103-194725") to [dev](https://console.bluemix.net/apps/9dcdc833-aedc-4fa5-b569-9605b28ed6ab?env_id=ibm:yp:us-south "ibm:yp:us-south:sk8325@nyu.edu:dev"), including commit 56a6646c98570b309444835535a01910d555c9b1

**Comment by shivkhattar** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/d97e2552-e5d6-4314-93cd-4603445a8066?env_id=ibm:yp:us-south "orders-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/de2c1c4b-d454-4678-8c83-87825f7d8ab3 "order-pipeline") deployed [orders](https://github.com/NYU-Devops-2020-Orders-Team/orders/releases/tag/deploy-ibm-yp-us-south-sk8325-nyu.edu-prod-20201116-013849 "deploy-ibm-yp-us-south-sk8325-nyu.edu-prod-20201116-013849") to [prod](https://console.bluemix.net/apps/72e50e75-2807-435f-b5d1-e24358ccc3c9?env_id=ibm:yp:us-south "ibm:yp:us-south:sk8325@nyu.edu:prod"), including commit 56a6646c98570b309444835535a01910d555c9b1

**Pull request merged**